README.md

```diff
@@ -98,12 +97,12 @@ Ukážeme si jednu z mnoha.
 
 Principy Docs-as-Code:
 
-- Správa verzí (Git)
-- Umožňuje aby na tom projektu spolupracovalo víc lidi
-- Kontrola změn kterou autor muže delegovat
-- Asynchronní spolupráce (každý, kdy má čas)
-- Automatické testování, generování, a publikace materiálů
-- Psaní ve značkovacích jazycích (Markdown, AsciiDoc, LateX, atd.) namísto WYSIWYG (LibreOffice, MS Word)
+- správa verzí (Git)
+- umožňuje, aby na projektu spolupracovalo víc lidi
+- kontrola změn, kterou autor může delegovat na spolupracovníky
+- asynchronní spolupráce (každý, kdy má čas)
+- automatické testování, generování, a publikace materiálů
+- psaní ve značkovacích jazycích (Markdown, AsciiDoc, LaTeX, atd.) namísto WYSIWYG (LibreOffice, MS Word)
```

Comment:
IBM Style Guide p. 20:
> Capitalization in lists
> Capitalize the ﬁrst word of each item in a vertical list.

> Example
> The routine makes the following conversions:
> * An EBCDIC value to a real number
> * A real number to an EBCDIC value
> * An EBCDIC value to an integer
> * An integer to an EBCDIC value
>
> In deﬁnition lists, capitalize the ﬁrst word of each term or descriptive name, unless it must be
> lowercase, and capitalize the ﬁrst word of each deﬁnition.
> Example
> All
>   You can perform all operations on resources.
> Create
>   You can view, edit, and add resources.
> Modify
>   You can view and edit resources.


Podle toho se ridim a vzdicky body kapitalizuju

Reply:
No jo, ale to platí pro angličtinu. V češtině se nekapitalizuje, pokud se nejedná o celou větu.

Reply:
https://www.krcmic.cz/odrazky-jaka-pravidla-dodrzovat/

Reply:
Ahjo, sorry nevedel jsem ze se to lisi v ruznych jazykach a brak IBM Style Guide moc vazne. Opravil jsem to zpet a moc dikuji za opravu.

Reply:
V pohodě. Schválně mě zajímá, jak to máte nastavené v ruštině? Zjistíš?

Reply:
Nikde jsem s tim nejak nepotkaval, proto musel googlit a zjistil jsem ze fungujou tam obycejne pravidla punktuace v jazyku. V odrazkech text se zacina s maleho pismena, na konci kazdeho radku muze byt tecka, carka, nebo strednik. Na konce posledniho radku vsak musi by tecka. V cislovanem seznamu totiz musi text se zacinat s velkeho pismena protoze je po tecke, napr. `1. Po jednicce je tecka`. Kdyz vsak po cislu je zavorka, tak musi text zacinat s malyho, napr. `1) po jednicce je zavorka`. Tady je dobry clanek s priklady: https://infogra.ru/typography/11-pravil-oformleniya-spiskov-i-perechnej, asi google umi text prelozit do cestiny.

Reply:
Zda se mi ze v rustine a cestine to je stejne

Reply:
Hehe, někdy jsou ty typos velmi vtipné jako například *brak IBM Style Guide*. Zajímalo by mě, co by si z toho vydedukoval Freud.